.changes/cli-plugin-add.md

```diff
@@ -0,0 +1,6 @@
+---
+'tauri-cli': 'patch:feat'
+'@tauri-apps/cli': 'patch:feat'
+---
+
+Add back `tauri plugin add` command as an aliast for `tauri add` command.
```

Comment:
@amrbashir You reverted this change (a18a3c1c), but you did not delete the change file.

Reply:
Thank you, will delete that one as well